app/components/article-featured-video.js

```diff
@@ -23,15 +25,27 @@ export default Component.extend(InViewportMixin,
 			'isVideoDrawerVisible:is-fixed',
 			'withinPortableInfobox:within-portable-infobox:without-portable-infobox'
 		],
-		hasTinyPlayIcon: computed.or('withinPortableInfobox', 'isVideoDrawerVisible'),
+
+		ads: inject.service(),
+		fastboot: inject.service(),
+		wikiVariables: inject.service(),
+
 		isPlayerLoading: true,
-		isPlaying: false,
+		autoplay: computed('withinPortableInfobox', function () {
+			return !this.get('fastboot.isFastBoot') &&
+				!this.get('withinPortableInfobox') &&
+				(system !== 'ios' || isSafariMinVer(10)) &&
+				$.cookie(autoplayCookieName) !== '0' &&
```

Comment:
I don't see where it's set, 
is it set on wikia.com domain? to make sure it sticks between wikis?